bob/src/cluster/quorum.rs

```diff
@@ -318,6 +319,91 @@ impl Quorum {
         }
         Ok(())
     }
+
+    async fn collect_remote_exists(
+        result: &mut [bool],
+        keys: &[BobKey],
+        indexes_by_node: &mut HashMap<Node, IndexMap>,
+    ) -> Result<(), Error> {
```

Comment:
Looks like this function does not return any error. It might be better to remove `Result<(), Error>` as return value